deploy/bz2068601/10-etcdDivergentRevisionsSRE.PrometheusRule.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: custom-etcd-prometheus-rules
+  namespace: openshift-etcd-operator
+spec:
+  groups:
+  - name: etcd
+    rules:
+    - alert: etcdDivergentRevisionsSRE
+      annotations:
+        description: etcd is reporting divergent member revisions,
+          with a standard deviation of {{ $value }}.  Members may be
+          stuck or, in extreme circumstances, split brained.
+        runbook_url: https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/FIXME
```

Comment:
We can optimisitically set `https://github.com/openshift/runbooks/blob/master/alerts/cluster-etcd-operator/etcdDivergentRevisionsSRE.md` here.  Or leave it as FIXME for now; I'm fine either way

Reply:
It won't matter, we generate an SOP link from the alert name.  I'll pair down to just a "message" annotation to be consistent with other alerts we manage.